tests/test_repository.py

```
@@ -334,3 +338,83 @@ def test_deepen_clone_for_diff_os_error(tmp_path: Path) -> None:
+            )
+
+
+def test_get_local_repo_info_returns_branch(tmp_path: Path) -> None:
+    from docsfy.repository import get_local_repo_info
+
+    with patch("docsfy.repository.subprocess.run") as mock_run:
+        mock_run.side_effect = [
+            MagicMock(returncode=0, stdout="abc123\n", stderr=""),
+            MagicMock(returncode=0, stdout="develop\n", stderr=""),
+        ]
+        repo_path, sha, branch = get_local_repo_info(tmp_path)
```

Ruff (0.15.5)

[warning] Unpacked variable `repo_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)